examples/dial/example_dial.cpp

```diff
@@ -16,9 +16,10 @@
 #include <string>
 #include <vector>
 
-#include "robot/client.hpp"
-#include "robot/service.hpp"
-#include "rpc/dial.hpp"
+#include <components/generic/client.hpp>
+#include <robot/client.hpp>
+#include <robot/service.hpp>
+#include <rpc/dial.hpp>
```

Comment:
[q] Why are we moving to system-first imports?

Reply:
Just needed to pick an idiom and stick to it, really. Everywhere under `src/` is following "first main include, then system headers, then third party, then our proto, then our code" so we should probably do the same here.

Reply:
Talked in person: We are moving to `<..>` imports for everything for stylistic reasons.